src/translations/en/hints.json

```diff
@@ -45,5 +45,11 @@
   "TEMPERATURE_LIMIT": "Will significantly decrease maximum power when threshold is reached.",
   "CURRENT_LIMIT": "Will hold the maximum possible RPM at the allowed current.",
   "SINE_MODE_POWER": "Fixed duty cycle level to be used when using sine mode. CAUTION: Values above 6 might damage motor and/or ESC.",
-  "BIDIRECTIONAL_MODE": "Neutral position is in the center of throttle. Spins in one direction when above center and in the other direction when below center position."
+  "BIDIRECTIONAL_MODE": "Neutral position is in the center of throttle. Spins in one direction when above center and in the other direction when below center position.",
+  "MOTOR_STARTUP_THROTTLE" : "Throttle setting to get motor spinning, 300 is about 15% throttle. [Max throttle: 2047]",
```

Comment:
Might make sense to Prefix with `GIL32_` so we have some context for translation